services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -180,6 +182,30 @@ export class NoticeOfWorkApplication extends Component {
     });
   };
 
+  renderOriginalValues = (path) => {
+    const prevValue = get(this.props.originalNoticeOfWork, path);
+    const currentValue = get(this.props.noticeOfWork, path);
+    // only show edited if user has updated the field
+    const isNewValue = isUndefined(prevValue) && !isNull(currentValue);
```

Comment:
Isn't `isNewValue = prevValue !== currentValue` enough and more correct? Why does the previous value have to be undefined and the current value can't be null? Data can be changed to null and that is considered a new value.

Reply:
ah, great question Luke! 

When a new activity is added, the fields are not registered in the initialValues, so by default they are undefined, and without that check - all new fields added to the form automatically say "edited" although they are brand new to the form. 

These are the situations I have identified: 
undefined - because they do not exist in the original application (isNewValue = prevValue !== currentValue works for this case)
undefined - because they are net new fields - the above check doesn't work in this situation as previous the check would be undefined !== null, which is true. So an additional check was added, does current === a value

Reply:
also, a good point that the current value can be changed back to null. 

The issue with that, is we do not capture any history. So if the original was 'null' || 'undefined', and the value updated to "directions to site", the FE would show "edited", but if the field was changed to null, we would be comparing the null to very first value of null/undefined which would show that the field wasn't edited. 

Following? There are so many edge cases with this one, but without history we have no way to capture if it was filled in and then changed back. 

Now if the original had a value, and is now null.... I dont handle that correctly. I'll add an additional case for that situation

Reply:
Sounds good. That sounds wrong that we don't capture any history if the value is set back to null.

Reply:
It does capture the audit history of the most current date/person. But it doesn't track every individual field, we only have access to original values and current values and the last time the entire form was edited.